infra/terraform/global/main.tf

```diff
@@ -48,3 +48,29 @@ module "log_pruning" {
       days: 30
 EOF
 }
+
+// Backup etcd volumes attached to kubernetes masters -->
+
+# Create Snapshot policy document
+data "template_file" "lambda_create_snapshot_policy" {
+  template = "${file("assets/create_etcd_ebs_snapshot/lambda_create_snapshot_policy.json")}"
```

Comment:
I think you can use the `aws_iam_policy_document` terraform resource (we used this in few other places before, [example from an old PR](https://github.com/ministryofjustice/analytics-platform-ops/pull/108/files#diff-cea909238eadd577d031fa1cef8241d6R12)).

Probably one of the reasons why we preferred that was as sometimes terraform was getting confused when diffing JSON blobs, but maybe it's not really a big deal to use a template file.

Reply:
I think this way is more succinct. Plus in it's raw form you can use it in more ways.  You can even just copy and paste from policy editors.  I use the policy-document resource when defining the assume-policy as it is short.  I always try and avoid heredocs for this.   I guess it's a style choice.